scriptmodules/emulators/aethersx2.sh

```diff
@@ -0,0 +1,530 @@
+#
+
+rp_module_id="aethersx2"
+rp_module_desc="PS2 emu - AetherSX2"
+rp_module_help="ROM Extensions: .iso .chd\n\nCopy your PS2 roms to $romdir/ps2 and the BIOs into /home/RetroPie/BIOS/ps2"
+rp_module_licence="PROP"
+rp_module_repo="git https://github.com/retropieuser/aethersx2.git main"
+rp_module_section="exp"
+rp_module_flags="!all 64bit"
+
+function depends_aethersx2() {
+    local depends=(matchbox-window-manager xorg xserver-xorg-input-all mesa-vulkan-drivers)
+
+    getDepends ${depends[@]}
+}
+
+function sources_aethersx2() {
+    gitPullOrClone
+}
+
+function install_aethersx2() {
+    tar -xzvf AetherSX2-v1.5-3606.tar.gz -C "/opt/retropie/emulators/aethersx2"
+    chmod +x /opt/retropie/emulators/aethersx2/AetherSX2-v1.5-3606/usr/bin/aethersx2
+}
+
+function configure_aethersx2() {
+    mkRomDir "ps2"
+    
+    local launch_prefix
+    isPlatform "kms" && launch_prefix="XINIT-WM:"
+
+    addEmulator 0 "$md_id" "ps2" "$launch_prefix$md_inst/AetherSX2-v1.5-3606/usr/bin/aethersx2 -nogui %ROM%"
+    addSystem "ps2"
+
+    [[ "$md_mode" == "remove" ]] && return
+
+    moveConfigDir "$home/.config/aethersx2" "$md_conf_root/ps2/Config"
+    mkUserDir "$md_conf_root/ps2/Config"
+    
+    # preset a few options on a first installation
+    if [[ ! -f "$md_conf_root/ps2/Config/inis/PCSX2.ini" ]]; then
+        cat >"$md_conf_root/ps2/Config/inis/PCSX2.ini" <<_EOF_
+[UI]
+SettingsVersion = 1
+InhibitScreensaver = true
+ConfirmShutdown = false
+StartPaused = false
+PauseOnFocusLoss = false
+StartFullscreen = true
+DoubleClickTogglesFullscreen = true
+HideMouseCursor = false
+RenderToSeparateWindow = false
+HideMainWindowWhenRunning = false
+DisableWindowResize = false
+Theme = darkfusion
+DisplayWindowGeometry = AdnQywADAAAAAAAAAAAAAAAAB38AAAQ3AAAABAAAABQAAAd7AAAEMwAAAAAAAAAAB4AAAAAEAAAAFAAAB3sAAAQz
+MainWindowGeometry = AdnQywADAAAAAAAAAAAAAAAAB38AAAQ3AAAABAAAABQAAAd7AAAEMwAAAAAAAAAAB4AAAAAEAAAAFAAAB3sAAAQz
+MainWindowState = AAAA/wAAAAD9AAAAAAAAB3gAAAP3AAAABAAAAAQAAAAIAAAACPwAAAABAAAAAgAAAAEAAAAOAHQAbwBvAGwAQgBhAHIAAAAAAP////8AAAAAAAAAAA==
+
+
+[Folders]
+Bios = ../../../../../../home/pi/RetroPie/BIOS/ps2
+Snapshots = snaps
+Savestates = sstates
+MemoryCards = memcards
+Logs = logs
+Cheats = cheats
+CheatsWS = cheats_ws
+CheatsNI = cheats_ni
+Cache = cache
+Textures = textures
+InputProfiles = inputprofiles
+
+
+[EmuCore]
+CdvdVerboseReads = false
+CdvdDumpBlocks = false
+CdvdShareWrite = false
+EnablePatches = true
+EnableCheats = false
+EnablePINE = false
+EnableWideScreenPatches = false
+EnableNoInterlacingPatches = false
+EnableRecordingTools = true
+EnableGameFixes = true
+SaveStateOnShutdown = false
+EnableDiscordPresence = false
+InhibitScreensaver = true
+ConsoleToStdio = false
+HostFs = false
+BackupSavestate = true
+SavestateZstdCompression = true
+McdEnableEjection = true
+McdFolderAutoManage = true
+WarnAboutUnsafeSettings = true
+GzipIsoIndexTemplate = $(f).pindex.tmp
```

Comment:
this line here is what is causing the error to come up

seems like the helper scripts dont like $(f)

Reply:
Yeah I think you're right here, I've created a BIOs symlink with BIOS/ps2 folder, so that'll probably help, have a try with this script but it should create the ini file ok now